feature_tests/features/product_team.failure.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Product Teams Failure Scenarios
+
+  Scenario: Failure - invalid Product Team Name
+    Given ODS Organisations:
+      | ods_code | name    |
+      | H8S7A    | BURENDO |
+    And Users:
+      | user_id          | name      |
+      | test@example.org | Test User |
+    When User "test@example.org" creates Product Team {00702d39-e65f-49f5-b9ef-6570245bfe17} called " " for supplier H8S7A
+    Then the operation was not successful
+    And the error is AssertionError
+
+  Scenario: Failure - unknown supplier
```

Comment:
This isn't testing domain logic -it's testing the test configuration (i.e. no domain code is called to create a product team)